sormas-ui/src/main/java/de/symeda/sormas/ui/importer/AbstractImportLayout.java

```diff
@@ -66,8 +76,19 @@ protected void addDownloadResourcesComponent(int step, ClassResource importGuide
 			CssStyles.VSPACE_TOP_3,
 			CssStyles.VSPACE_2);
 
-		FileDownloader dataDictionaryDownloader = new FileDownloader(dataDictionaryResource);
-		dataDictionaryDownloader.extend(dataDictionaryButton);
+		new FileDownloader(new StreamResource(() -> new DownloadUtil.DelayedInputStream((out) -> {
+			try {
+				String documentPath = FacadeProvider.getInfoFacade().generateDataDictionary();
+				IOUtils.copy(Files.newInputStream(new File(documentPath).toPath()), out);
+			} catch (IOException e) {
+				LoggerFactory.getLogger(AboutView.class).error("Failed to generate data dictionary", e);
+
+				// fall back to pre-generated document
+				InputStream preGeneratedDocumentStream = new ClassResource("/doc/SORMAS_Data_Dictionary.xlsx").getStream().getStream();
+				IOUtils.copy(preGeneratedDocumentStream, out);
+			}
+		}, (e) -> {
+		}), createFileNameWithCurrentDate(ExportEntityName.DATA_DICTIONARY, ".xlsx"))).extend(dataDictionaryButton);
```

Comment:
This is duplicate to parts of `AboutView.createDocumentsSection()`; It would be better to move it to a seperate method, e.g. `DownloadUtil.attachDataDictionaryDownloader(Button button)`